packages/ui-components/src/components/modal/modal.tsx

```diff
@@ -16,7 +16,7 @@ export class KvModal implements IModalConfig, IModalEvents {
 	/** @inheritdoc */
 	@Prop() hasBackdrop: boolean = true;
 	/** @inheritdoc */
-	@Prop() closable?: boolean = true;
+	@Prop() closeOnOverlayClick?: boolean = true;
```

Comment:
Is there a use case where you don't what this to happen? I mean, clicking on the overlay and the modal doesn't close?

Also, I don't this is the prop you want to change. This prop is related to if the component is closable by either clicking on the overlay or by clicking on the X button.

Reply:
If your intention here is to reuse the modal component to create a "wizard" flow and you want to prevent the user from accidentally closing this modal by clicking on the overlay container, then I don't think that the overlay should exist because it will have different behaviours on different modals. But, it's not a problem with your approach I guess.

Just please coherency, do the following:
- `hasBackdrop` should be `showOverlay` in order to be coherent with the CSS class and the `showCloseButton` prop.

Reply:
makes sense, changed it :)